doc/source/conf.py

```diff
@@ -115,7 +115,10 @@
 # Fix excessive margins in mermaid output.
 # See issue: https://github.com/mermaid-js/mermaid/issues/1800#issuecomment-741617143
 mermaid_output_format = "png"
-mermaid_params = ["--width", "2000"]
+mermaid_params = ["--width", "2000", "--backgroundColor", "white"]
+
```

Comment:
Do we still need this even though we are passing `bgcolor="white"` to the images? If not, maybe just revert it
```suggestion
mermaid_params = ["--width", "2000"]

```

Reply:
Let us force white backgrounds for both mermaid and graphviz diagrams 🚀